openmetadata-airflow-apis/tests/unit/test_helpers.py

```diff
@@ -0,0 +1,31 @@
+#  Copyright 2021 Collate
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#  http://www.apache.org/licenses/LICENSE-2.0
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+"""
+Test helper functions
+"""
+from unittest import TestCase
```

Comment:
nit: we could follow the same folder structure for the tests as the one we have in `src`.

Reply:
Agree. I'll fix this on the Airflow 2.3.3 PR, as it modifies everything

Thanks for the comment 🙏